frontend/src/pages/PendingAssessments/RequestItem/RequestButtons/RequestButtons.tsx

```diff
@@ -0,0 +1,29 @@
+import { FC } from 'react';
+
+import { buttonSpinner } from 'animations';
+import ButtonLoader from 'components/ButtonLoader';
+import { ButtonLabels } from 'constants/ButtonLabels';
+import { IAssessmentRequestButtonsProps } from 'pages/PendingAssessments/types';
+
+import { ActionButton, ButtonsContainer } from './styled';
+
+const RequestButtons: FC<IAssessmentRequestButtonsProps> = ({
+  approveAssessment,
+  declineAssessment,
+  isActionLoading,
+}) => (
+  <ButtonsContainer item xs={3} rowSpacing={1}>
+    <ActionButton
+      variant={isActionLoading ? 'mediumOutlined' : 'mediumContained'}
+      onClick={approveAssessment}
+      disabled={isActionLoading}
```

Comment:
what is the rule about props order? @ostapdans99

Reply:
@irina-sinitsyna https://docs.google.com/document/d/1Gl7Rs2KoH1wkLhQongugUwKcjK80ozljB4cFCsUdeqE/edit#heading=h.sq0sznirkfdr. We came to the conclusion that it would be better at first specify attribute properties like: disabled, type, variant, ... . Then, as in the 7th paragraph of the documentation

Reply:
required -> optional,
info -> bool -> handler

according to this rule variant should be at the first place, then disabled, then onClick, isn't it?

Reply:
required -> optional,
info -> bool -> handler

according to this rule variant should be at the first place, then disabled, then onClick, isn't it?

@ostapdans99

Reply:
done

Reply:
@irina-sinitsyna you are right, my mistake